great_expectations/datasource/fluent/pandas_datasource.py

```diff
@@ -124,6 +124,7 @@ def get_batch_list_from_batch_request(
                 exclude=self._EXCLUDE_FROM_READER_OPTIONS,
                 exclude_unset=True,
                 by_alias=True,
+                config_provider=self._datasource._config_provider,
             ),
```

Comment:
@billdirks here a `DataAsset` is resolving any `ConfigStr` variables to pass to the `BatchSpec`